Update protocol update block number on various contents (#420)
klaytn/klaytn-docs: docs/klaytn/design/computation/computation-cost/computation-cost.md

The below table shows the computation cost of EVM opcodes. The computation cost was determined based on experiments.

{% hint style="success" %}

> NOTE: Computation costs have been changed with `IstanbulEVM` protocol upgrade, or the "hard fork".

Reviewer: ```suggestion
NOTE: Computation costs have been changed with the `IstanbulEVM` protocol upgrade, or the "hard fork".
```

If you want the previous document, please refer to [previous document](computation-cost-previous.md).

> `IstanbulEVM` protocol upgrade block number is like the followings.

Reviewer: ```suggestion
`IstanbulEVM` protocol upgrade block number is as follows.
```
